run_sets/testsuite.yml

```diff
@@ -43,6 +43,8 @@
 - features/salt.feature
 - features/salt_minions_page.feature
 - features/salt_minion_details.feature
+- features/salt_download_endpoint.feature
```

Comment:
@meaksh  can you put the feature salt randomly ordered in idempotent? so we can assume that they are idempotent and not order dipendent 🎏